cache/apply.go

```diff
@@ -85,16 +87,6 @@ func (s *RawProviderServer) ApplyResourceChange(ctx context.Context, req *tfprot
 		s.logger.Trace("[ApplyResourceChange]", "[PlannedState]", dump(plannedState))
 
 		resp.NewState = &plannedState
-	case !applyPlannedState.IsNull() && !applyPriorState.IsNull():
```

Comment:
This is the "update" condition, which previously should never be hit. With `keepers` its a valid condition, so remove the error.